### PR TITLE
LPS-70110 semantic versioning

### PR DESCRIPTION
**Pull request by pavel-savinov**

### CODE REVIEW

**Comment by pavel-savinov** [comment]
#46211
Attribute name moved to WebKeys.

**Comment by pavel-savinov** [comment]
ci:retest

**Comment by pavel-savinov** [comment]
ci:retest

File: modules/apps/foundation/map/map-api/src/main/java/com/liferay/map/constants/MapProviderWebKeys.java [hunk 1]
```diff
@@ -19,6 +19,9 @@
  */
 public class MapProviderWebKeys {
 
+	public static final String MAP_PROVIDER_CONFIGURATINO_PREFIX =
+		"MAP_PROVIDER_CONFIGURATINO_PREFIX";
```
Comment (jbalsas) [inline modules/apps/foundation/map/map-api/src/main/java/com/liferay/map/constants/MapProviderWebKeys.java]: There's a typo here: MAP_PROVIDER_CONFIGURATI~NO~ON_PREFIX

**Comment by brianchandotcom** [comment]
Please resend:

```
CONFIGURATINO
```

Thx.

**Pull request closed without merging**